### ISSUE TITLE
batchsize affects the quality of reconstructed images when training dvae

### ISSUE THREAD

**Issue by CCchenxiaoxue** [lucidrains/DALLE-pytorch#440]
when I set batch=512, the quality of reconstructed images is worse compared with batch=8, does anyone know how to fix it? thank you~

**Comment by alexriedel1** [comment]
Batch size and learning rate should be adjusted accordingly to each other. For optimizers like Adam it may not be that important but try to reduce your learning rate when you increase your batch size and vice versa
